jiminy/src/lib.rs

```diff
@@ -46,3 +49,25 @@ impl<I: Display + Into<ProgramError>, P: Display + Into<ProgramError>>
         }
     }
 }
+
+#[allow(deprecated)]
+pub struct RemoveLiqQuoteProgErr<I, P>(pub RemoveLiqQuoteErr<I, P>);
+
+impl<I: Display + Into<ProgramError>, P: Display + Into<ProgramError>>
+    From<RemoveLiqQuoteProgErr<I, P>> for ProgramError
+{
+    #[allow(deprecated)]
+    fn from(RemoveLiqQuoteProgErr(e): RemoveLiqQuoteProgErr<I, P>) -> Self {
+        let msg = e.to_string();
+        sol_log(&msg);
+        match e {
+            RemoveLiqQuoteErr::Overflow => Inf1CtlCustomProgErr(Inf1CtlErr::MathError).into(),
+            RemoveLiqQuoteErr::NotEnougLiquidity(_) => {
```

Comment:
nit: could you just fix the typo in this PR too? Thanks
```suggestion
            RemoveLiqQuoteErr::NotEnoughLiquidity(_) => {
```